.github/workflows/publish_layer.yml

```diff
@@ -0,0 +1,79 @@
+name: Deploy layer to all regions
+
+permissions:
+  id-token: write
+  contents: read
+
+on:
+  # TODO: remove after setup is finished
```

Comment:
```suggestion
```

Removing as I actually had misread by in the other workflow. There could be scenarios where we might want to trigger it manually (hotfix) in the far future.